servant/src/Servant/API/ResponseHeaders.hs

```diff
@@ -65,6 +68,18 @@ data Headers ls a = Headers { getResponse :: a
 instance (NFDataHList ls, NFData a) => NFData (Headers ls a) where
     rnf (Headers x hdrs) = rnf x `seq` rnf hdrs
 
+instance {-# OVERLAPPING #-} MimeRender PlainText a => MimeRender PlainText (Headers _ls a) where
```

Comment:
How are these related to `UVerb`?  Of why did yo introduce them?  Having default non-overlappable instances seems problematic, but I can't think of a reason why anybody would want to overlap these, so maybe it's fine?

Reply:
As far as I understand, they are needed because the `HasServer` instance for `UVerb ... as` has an `AllCTRender` on the elements of `as`, and I don't see how this could be changed to a constraint on the types inside potential `Headers h` wrappers.

It is possible to get away without these instances in the case of `Verb`, because there is a special instance for `HasServer (Verb ... (Headers h a))` that has an `AllCTRender` constraint on `a` (rather than the whole `Headers h a`).

Reply:
Ah, I see.  This LGTM then.